assets/src/scss/layout/navbar/_search.scss

```diff
@@ -165,7 +165,7 @@
 .nav-search-toggle-container {
   display: none;
   height: $menu-height-small;
-  padding: 8px 12px;
+  padding: #{$sp-1} #{$sp-1x};
```

Comment:
You should be able to use directly `$sp-1` here 🤔

Reply:
There is also a nice syntax for this called numbers :wink: